keyboards/ergodox_ez/config.h

```diff
@@ -78,14 +94,20 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 // #define RGB_MIDI
 #define RGBW_BB_TWI
 
+#ifdef RGBW_BB_TWI
+#if (defined(ERGODOX_LED_30) + defined(ERGODOX_LED_15) + defined(ERGODOX_LED_15_MIRROR)) != 1
+#error "You must define exactly one of the ERGODOX_LED options."
+#endif
+#endif
+
 #define RGBW 1
 
 /* Set 0 if debouncing isn't needed */
 #define DEBOUNCE    15
 
 #define PREVENT_STUCK_MODIFIERS
 
-#define USB_MAX_POWER_CONSUMPTION 500
+#define USB_MAX_POWER_CONSUMPTION 400
```

Comment:
Does this change have something to do with the rest of the PR?

Reply:
i don't think it did, and i honestly have no idea how that happened.

Reply:
Could this be reverted then? 
```suggestion
#define USB_MAX_POWER_CONSUMPTION 500
```